### DISCUSSION TITLE
after upgrade to 0.147 error 500 column scripts_script run_as_user doesn't exist

### DISCUSSION THREAD

**Discussion by Alainr69** [amidaware/tacticalrmm#1260] [Q&A]
**Server Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04, Debian 10]
 - Browser: [e.g. chrome, safari]
 - RMM Version (as shown in top left of web UI):

**Installation Method:**
  - [x] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI):
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
Error Django db after update

![django-db-error](https://user-images.githubusercontent.com/109462705/186234717-b6f6136a-50f1-4dbd-9778-bb6e56c592ec.PNG)

**Comment by wh1te909** [comment]
what OS are you on. what version did you upgrade from? post entire output of the update script

**Reply by wh1te909** [reply]
run update script again with the `--force` flag check docs for how to do this and paste output
